src/api/mod.rs

```diff
@@ -19,13 +20,16 @@ pub struct Command<T: StructOpt> {
 pub enum ApiCommand {
     SigningKeys(signing_keys::SigningKeysCommand),
     Users(users::UsersCommand),
+    #[structopt(flatten)]
+    Update(update::UpdateCommand),
```

Comment:
If this impacts the order the commands are listed in `--help` lets keep it alphabetical.

Command should also be `Upgrade` not `Update` - will have to change that in a couple spots across the PR but only mentioning it here.